modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -103,6 +103,32 @@ public static double Acos(double s)
             return Math.Acos(s);
         }
 
+        /// <summary>
+        /// Gets the difference between two angles (in radians) by a normalized value,
+        /// but wraps around <see cref="Tau"/>, also creating a shortest path.
+        /// </summary>
```

Comment:
The C# documentation is different from the GDScript documentation. We try to keep them in sync.

Reply:
Wrote a new doc entirely and copied it over to C#.